templates/lead.html

```diff
@@ -0,0 +1,12 @@
+<b>{{lead.display_name}}</b><br/>
+{% if lead.contacts %}
+  <b>Contacts:</b> {% if lead.contacts[0].name %}{{lead.contacts[0].name}}{% else %}Unnamed{% endif %}
+  {% if lead.contacts|length > 1 %}+{{ lead.contacts|length - 1}}{% endif %}<br/>
+{% endif %}
+<b>Status:</b> {{lead.status_label}}<br/>
+{% if lead.opportunities %}
+<b>Opportunities:</b>
```

Comment:
you can just drop this label altogether. it's obvious by the word 'opportunity' below what it is